Runtime/AvatarCreator/Scripts/UI/Buttons/AvatarListItem.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using ReadyPlayerMe.AvatarCreator;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+public class AvatarListItem : MonoBehaviour
```

Comment:
I think a better name for this class would be AvatarButton or AvatarSelectButton.